[2019-11-02 09:48:30] console.DEBUG: Command "server:start" exited with code "1" {"command":"server:start","code":1} []
[2019-11-02 09:48:42] php.DEBUG: Warning: fsockopen(): unable to connect to 127.0.0.1:8001 (Une tentative de connexion a échoué car le parti connecté n’a pas répondu convenablement au-delà d’une certaine durée ou une connexion établie a échoué car l’hôte de connexion n’a pas répondu. ) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"C:\\\\Users\\\\jean\\\\Desktop\\\\dotdev_lib\\\\symfony\\\\vendor\\\\symfony\\\\web-server-bundle\\\\WebServerConfig.php\",\"line\":141,\"trace\":[{\"file\":\"C:\\\\Users\\\\jean\\\\Desktop\\\\dotdev_lib\\\\symfony\\\\vendor\\\\symfony\\\\web-server-bundle\\\\WebServerConfig.php\",\"line\":54,\"function\":\"findBestPort\",\"class\":\"Symfony\\\\Bundle\\\\WebServerBundle\\\\WebServerConfig\",\"type\":\"->\"}],\"count\":1})"} []
[2019-11-02 09:49:08] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\http-kernel\EventListener\RouterListener.php line 138 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:138, Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:182)"} []
[2019-11-02 09:49:10] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"9bfba4"},"request_uri":"http://localhost:8001/_wdt/9bfba4","method":"GET"} []
[2019-11-02 09:49:11] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost:8001/")" at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\http-kernel\EventListener\RouterListener.php line 138 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost:8001/\") at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:138, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:70)"} []
[2019-11-02 10:06:40] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\http-kernel\EventListener\RouterListener.php line 138 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:138, Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:182)"} []
[2019-11-02 10:06:41] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"dd2d3c"},"request_uri":"http://localhost:8001/_wdt/dd2d3c","method":"GET"} []
[2019-11-02 10:08:21] console.DEBUG: Command "server:stop" exited with code "1" {"command":"server:stop","code":1} []
[2019-11-02 10:08:32] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\http-kernel\EventListener\RouterListener.php line 138 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:138, Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:182)"} []
[2019-11-02 10:08:32] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"523e2b"},"request_uri":"http://localhost:8001/_wdt/523e2b","method":"GET"} []
[2019-11-02 10:09:33] console.DEBUG: Command "server:status" exited with code "1" {"command":"server:status","code":1} []
[2019-11-02 10:19:17] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\http-kernel\EventListener\RouterListener.php line 138 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:138, Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:182)"} []
[2019-11-02 10:19:18] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"16a4bf"},"request_uri":"http://localhost:8001/_wdt/16a4bf","method":"GET"} []
[2019-11-02 10:19:28] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_route":"_profiler","_controller":"web_profiler.controller.profiler::panelAction","token":"16a4bf"},"request_uri":"http://localhost:8001/_profiler/16a4bf?panel=request","method":"GET"} []
[2019-11-02 10:19:28] request.INFO: Matched route "_profiler_search_bar". {"route":"_profiler_search_bar","route_parameters":{"_route":"_profiler_search_bar","_controller":"web_profiler.controller.profiler::searchBarAction"},"request_uri":"http://localhost:8001/_profiler/search_bar?panel=request","method":"GET"} []
[2019-11-02 10:28:10] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\http-kernel\EventListener\RouterListener.php line 138 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:138, Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:182)"} []
[2019-11-02 10:28:10] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"2fab7e"},"request_uri":"http://localhost:8002/_wdt/2fab7e","method":"GET"} []
[2019-11-02 10:28:10] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost:8002/")" at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\http-kernel\EventListener\RouterListener.php line 138 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost:8002/\") at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:138, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:70)"} []
[2019-11-02 10:54:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\http-kernel\EventListener\RouterListener.php line 138 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:138, Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:182)"} []
[2019-11-02 10:54:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"13b293"},"request_uri":"http://127.0.0.1:8001/_wdt/13b293","method":"GET"} []
[2019-11-02 10:54:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://127.0.0.1:8001/")" at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\http-kernel\EventListener\RouterListener.php line 138 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://127.0.0.1:8001/\") at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:138, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:70)"} []
[2019-11-02 10:54:18] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\http-kernel\EventListener\RouterListener.php line 138 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:138, Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:182)"} []
[2019-11-02 10:54:18] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"1f9e4f"},"request_uri":"http://127.0.0.1:8001/_wdt/1f9e4f","method":"GET"} []
[2019-11-02 11:49:57] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\http-kernel\EventListener\RouterListener.php line 138 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:138, Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:182)"} []
[2019-11-02 11:49:58] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"87737e"},"request_uri":"http://127.0.0.1:8001/_wdt/87737e","method":"GET"} []
[2019-11-02 11:53:28] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\PagesController::index"},"request_uri":"http://127.0.0.1:8001/","method":"GET"} []
[2019-11-02 11:53:28] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-02 11:53:28] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable. Controller "App\Controller\PagesController" does neither exist as service nor as class" at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\http-kernel\Controller\ControllerResolver.php line 88 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable. Controller \"App\\Controller\\PagesController\" does neither exist as service nor as class at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\http-kernel\\Controller\\ControllerResolver.php:88)"} []
[2019-11-02 11:53:29] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"77e930"},"request_uri":"http://127.0.0.1:8001/_wdt/77e930","method":"GET"} []
[2019-11-02 11:58:03] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\PagesController::index"},"request_uri":"http://127.0.0.1:8001/","method":"GET"} []
[2019-11-02 11:58:03] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-02 11:58:03] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable. Controller "App\Controller\PagesController" does neither exist as service nor as class" at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\http-kernel\Controller\ControllerResolver.php line 88 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable. Controller \"App\\Controller\\PagesController\" does neither exist as service nor as class at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\http-kernel\\Controller\\ControllerResolver.php:88)"} []
[2019-11-02 11:58:04] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"0a7ede"},"request_uri":"http://127.0.0.1:8001/_wdt/0a7ede","method":"GET"} []
[2019-11-02 11:58:31] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\PagesController::index"},"request_uri":"http://127.0.0.1:8002/","method":"GET"} []
[2019-11-02 11:58:31] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-02 11:58:32] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable. Controller "App\Controller\PagesController" does neither exist as service nor as class" at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\http-kernel\Controller\ControllerResolver.php line 88 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable. Controller \"App\\Controller\\PagesController\" does neither exist as service nor as class at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\http-kernel\\Controller\\ControllerResolver.php:88)"} []
[2019-11-02 11:58:32] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"876b68"},"request_uri":"http://127.0.0.1:8002/_wdt/876b68","method":"GET"} []
[2019-11-02 12:01:57] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\PagesController::index"},"request_uri":"http://localhost:8001/","method":"GET"} []
[2019-11-02 12:01:57] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-02 12:01:57] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable. Controller "App\Controller\PagesController" does neither exist as service nor as class" at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\http-kernel\Controller\ControllerResolver.php line 88 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable. Controller \"App\\Controller\\PagesController\" does neither exist as service nor as class at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\http-kernel\\Controller\\ControllerResolver.php:88)"} []
[2019-11-02 12:01:58] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"192aa9"},"request_uri":"http://localhost:8001/_wdt/192aa9","method":"GET"} []
[2019-11-02 12:03:27] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\PagesController::index"},"request_uri":"http://localhost:8001/","method":"GET"} []
[2019-11-02 12:03:27] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-02 12:03:27] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable. Controller "App\Controller\PagesController" does neither exist as service nor as class" at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\http-kernel\Controller\ControllerResolver.php line 88 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable. Controller \"App\\Controller\\PagesController\" does neither exist as service nor as class at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\http-kernel\\Controller\\ControllerResolver.php:88)"} []
[2019-11-02 12:03:27] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"bb95e7"},"request_uri":"http://localhost:8001/_wdt/bb95e7","method":"GET"} []
[2019-11-02 12:05:59] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Config\Exception\LoaderLoadException: "The routing file "C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes.yaml" contains unsupported keys for "index": "default". Expected one of: "resource", "type", "prefix", "path", "host", "schemes", "methods", "defaults", "requirements", "options", "condition", "controller", "name_prefix", "trailing_slash_on_root", "locale", "format", "utf8" in C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes.yaml (which is loaded in resource "C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes.yaml")." at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\config\Loader\FileLoader.php line 166 {"exception":"[object] (Symfony\\Component\\Config\\Exception\\LoaderLoadException(code: 0): The routing file \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\" contains unsupported keys for \"index\": \"default\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\", \"controller\", \"name_prefix\", \"trailing_slash_on_root\", \"locale\", \"format\", \"utf8\" in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml (which is loaded in resource \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\"). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\config\\Loader\\FileLoader.php:166, InvalidArgumentException(code: 0): The routing file \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\" contains unsupported keys for \"index\": \"default\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\", \"controller\", \"name_prefix\", \"trailing_slash_on_root\", \"locale\", \"format\", \"utf8\". at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\routing\\Loader\\YamlFileLoader.php:271)"} []
[2019-11-02 12:05:59] request.CRITICAL: Exception thrown when handling an exception (Twig\Error\RuntimeError: An exception has been thrown during the rendering of a template ("The routing file "C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes.yaml" contains unsupported keys for "index": "default". Expected one of: "resource", "type", "prefix", "path", "host", "schemes", "methods", "defaults", "requirements", "options", "condition", "controller", "name_prefix", "trailing_slash_on_root", "locale", "format", "utf8" in C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes.yaml (which is loaded in resource "C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes.yaml")."). at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\twig-bundle\Resources\views\Exception\trace.html.twig line 13) {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"The routing file \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\" contains unsupported keys for \"index\": \"default\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\", \"controller\", \"name_prefix\", \"trailing_slash_on_root\", \"locale\", \"format\", \"utf8\" in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml (which is loaded in resource \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\").\"). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\twig-bundle\\Resources\\views\\Exception\\trace.html.twig:13, Symfony\\Component\\Config\\Exception\\LoaderLoadException(code: 0): The routing file \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\" contains unsupported keys for \"index\": \"default\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\", \"controller\", \"name_prefix\", \"trailing_slash_on_root\", \"locale\", \"format\", \"utf8\" in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml (which is loaded in resource \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\"). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\config\\Loader\\FileLoader.php:166, InvalidArgumentException(code: 0): The routing file \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\" contains unsupported keys for \"index\": \"default\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\", \"controller\", \"name_prefix\", \"trailing_slash_on_root\", \"locale\", \"format\", \"utf8\". at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\routing\\Loader\\YamlFileLoader.php:271)"} []
[2019-11-02 12:05:59] php.CRITICAL: Uncaught Exception: An exception has been thrown during the rendering of a template ("The routing file "C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes.yaml" contains unsupported keys for "index": "default". Expected one of: "resource", "type", "prefix", "path", "host", "schemes", "methods", "defaults", "requirements", "options", "condition", "controller", "name_prefix", "trailing_slash_on_root", "locale", "format", "utf8" in C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes.yaml (which is loaded in resource "C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes.yaml")."). {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"The routing file \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\" contains unsupported keys for \"index\": \"default\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\", \"controller\", \"name_prefix\", \"trailing_slash_on_root\", \"locale\", \"format\", \"utf8\" in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml (which is loaded in resource \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\").\"). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\twig-bundle\\Resources\\views\\Exception\\trace.html.twig:13, Symfony\\Component\\Config\\Exception\\LoaderLoadException(code: 0): The routing file \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\" contains unsupported keys for \"index\": \"default\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\", \"controller\", \"name_prefix\", \"trailing_slash_on_root\", \"locale\", \"format\", \"utf8\" in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml (which is loaded in resource \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\"). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\config\\Loader\\FileLoader.php:166, InvalidArgumentException(code: 0): The routing file \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\" contains unsupported keys for \"index\": \"default\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\", \"controller\", \"name_prefix\", \"trailing_slash_on_root\", \"locale\", \"format\", \"utf8\". at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\routing\\Loader\\YamlFileLoader.php:271, Symfony\\Component\\Config\\Exception\\LoaderLoadException(code: 0): The routing file \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\" contains unsupported keys for \"index\": \"default\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\", \"controller\", \"name_prefix\", \"trailing_slash_on_root\", \"locale\", \"format\", \"utf8\" in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml (which is loaded in resource \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\"). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\config\\Loader\\FileLoader.php:166, InvalidArgumentException(code: 0): The routing file \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\" contains unsupported keys for \"index\": \"default\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\", \"controller\", \"name_prefix\", \"trailing_slash_on_root\", \"locale\", \"format\", \"utf8\". at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\routing\\Loader\\YamlFileLoader.php:271)"} []
[2019-11-02 12:06:03] request.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "An exception has been thrown during the rendering of a template ("The routing file "C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes.yaml" contains unsupported keys for "index": "default". Expected one of: "resource", "type", "prefix", "path", "host", "schemes", "methods", "defaults", "requirements", "options", "condition", "controller", "name_prefix", "trailing_slash_on_root", "locale", "format", "utf8" in C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes.yaml (which is loaded in resource "C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes.yaml").")." at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\twig-bundle\Resources\views\Exception\trace.html.twig line 13 {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"The routing file \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\" contains unsupported keys for \"index\": \"default\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\", \"controller\", \"name_prefix\", \"trailing_slash_on_root\", \"locale\", \"format\", \"utf8\" in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml (which is loaded in resource \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\").\"). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\twig-bundle\\Resources\\views\\Exception\\trace.html.twig:13, Symfony\\Component\\Config\\Exception\\LoaderLoadException(code: 0): The routing file \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\" contains unsupported keys for \"index\": \"default\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\", \"controller\", \"name_prefix\", \"trailing_slash_on_root\", \"locale\", \"format\", \"utf8\" in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml (which is loaded in resource \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\"). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\config\\Loader\\FileLoader.php:166, InvalidArgumentException(code: 0): The routing file \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\" contains unsupported keys for \"index\": \"default\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\", \"controller\", \"name_prefix\", \"trailing_slash_on_root\", \"locale\", \"format\", \"utf8\". at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\routing\\Loader\\YamlFileLoader.php:271, Symfony\\Component\\Config\\Exception\\LoaderLoadException(code: 0): The routing file \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\" contains unsupported keys for \"index\": \"default\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\", \"controller\", \"name_prefix\", \"trailing_slash_on_root\", \"locale\", \"format\", \"utf8\" in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml (which is loaded in resource \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\"). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\config\\Loader\\FileLoader.php:166, InvalidArgumentException(code: 0): The routing file \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\" contains unsupported keys for \"index\": \"default\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\", \"controller\", \"name_prefix\", \"trailing_slash_on_root\", \"locale\", \"format\", \"utf8\". at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\routing\\Loader\\YamlFileLoader.php:271)"} []
[2019-11-02 12:06:03] request.CRITICAL: Exception thrown when handling an exception (Twig\Error\RuntimeError: An exception has been thrown during the rendering of a template ("The routing file "C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes.yaml" contains unsupported keys for "index": "default". Expected one of: "resource", "type", "prefix", "path", "host", "schemes", "methods", "defaults", "requirements", "options", "condition", "controller", "name_prefix", "trailing_slash_on_root", "locale", "format", "utf8" in C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes.yaml (which is loaded in resource "C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes.yaml")."). at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\twig-bundle\Resources\views\Exception\trace.html.twig line 13) {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"The routing file \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\" contains unsupported keys for \"index\": \"default\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\", \"controller\", \"name_prefix\", \"trailing_slash_on_root\", \"locale\", \"format\", \"utf8\" in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml (which is loaded in resource \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\").\"). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\twig-bundle\\Resources\\views\\Exception\\trace.html.twig:13, Symfony\\Component\\Config\\Exception\\LoaderLoadException(code: 0): The routing file \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\" contains unsupported keys for \"index\": \"default\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\", \"controller\", \"name_prefix\", \"trailing_slash_on_root\", \"locale\", \"format\", \"utf8\" in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml (which is loaded in resource \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\"). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\config\\Loader\\FileLoader.php:166, InvalidArgumentException(code: 0): The routing file \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes.yaml\" contains unsupported keys for \"index\": \"default\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\", \"controller\", \"name_prefix\", \"trailing_slash_on_root\", \"locale\", \"format\", \"utf8\". at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\routing\\Loader\\YamlFileLoader.php:271)"} []
[2019-11-02 12:06:33] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\PagesController::index"},"request_uri":"http://localhost:8001/","method":"GET"} []
[2019-11-02 12:06:33] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-02 12:06:33] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable. Controller "App\Controller\PagesController" does neither exist as service nor as class" at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\http-kernel\Controller\ControllerResolver.php line 88 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable. Controller \"App\\Controller\\PagesController\" does neither exist as service nor as class at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\http-kernel\\Controller\\ControllerResolver.php:88)"} []
[2019-11-02 12:06:33] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"48d244"},"request_uri":"http://localhost:8001/_wdt/48d244","method":"GET"} []
[2019-11-02 12:08:08] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\PagesController::index"},"request_uri":"http://localhost:8001/","method":"GET"} []
[2019-11-02 12:08:08] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-02 12:08:08] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable. Controller "App\Controller\PagesController" does neither exist as service nor as class" at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\http-kernel\Controller\ControllerResolver.php line 88 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable. Controller \"App\\Controller\\PagesController\" does neither exist as service nor as class at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\http-kernel\\Controller\\ControllerResolver.php:88)"} []
[2019-11-02 12:08:08] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"9eeea0"},"request_uri":"http://localhost:8001/_wdt/9eeea0","method":"GET"} []
[2019-11-02 12:08:12] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\PagesController::index"},"request_uri":"http://localhost:8002/","method":"GET"} []
[2019-11-02 12:08:12] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-02 12:08:12] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable. Controller "App\Controller\PagesController" does neither exist as service nor as class" at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\http-kernel\Controller\ControllerResolver.php line 88 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable. Controller \"App\\Controller\\PagesController\" does neither exist as service nor as class at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\http-kernel\\Controller\\ControllerResolver.php:88)"} []
[2019-11-02 12:08:12] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"9d499d"},"request_uri":"http://localhost:8002/_wdt/9d499d","method":"GET"} []
[2019-11-02 12:10:14] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\PagesController::index"},"request_uri":"http://localhost:8002/","method":"GET"} []
[2019-11-02 12:10:14] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-02 12:10:14] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable. Controller "App\Controller\PagesController" does neither exist as service nor as class" at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\http-kernel\Controller\ControllerResolver.php line 88 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable. Controller \"App\\Controller\\PagesController\" does neither exist as service nor as class at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\http-kernel\\Controller\\ControllerResolver.php:88)"} []
[2019-11-02 12:10:15] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"80f1bf"},"request_uri":"http://localhost:8002/_wdt/80f1bf","method":"GET"} []
[2019-11-02 12:12:03] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","controller":"App\\Controller\\PagesController::index"},"request_uri":"http://localhost:8002/","method":"GET"} []
[2019-11-02 12:12:03] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-02 12:12:03] request.WARNING: Unable to look for the controller as the "_controller" parameter is missing. [] []
[2019-11-02 12:12:03] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "Unable to find the controller for path "/". The route is wrongly configured." at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\http-kernel\HttpKernel.php line 135 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Unable to find the controller for path \"/\". The route is wrongly configured. at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\http-kernel\\HttpKernel.php:135)"} []
[2019-11-02 12:12:03] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"0f3174"},"request_uri":"http://localhost:8002/_wdt/0f3174","method":"GET"} []
[2019-11-02 12:13:10] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\PagesController::index"},"request_uri":"http://localhost:8002/","method":"GET"} []
[2019-11-02 12:13:10] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-02 12:13:10] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable. Controller "App\Controller\PagesController" does neither exist as service nor as class" at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\http-kernel\Controller\ControllerResolver.php line 88 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable. Controller \"App\\Controller\\PagesController\" does neither exist as service nor as class at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\http-kernel\\Controller\\ControllerResolver.php:88)"} []
[2019-11-02 12:13:11] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"804dac"},"request_uri":"http://localhost:8002/_wdt/804dac","method":"GET"} []
[2019-11-02 12:13:34] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_route":"_profiler","_controller":"web_profiler.controller.profiler::panelAction","token":"804dac"},"request_uri":"http://localhost:8002/_profiler/804dac?panel=logger","method":"GET"} []
[2019-11-02 12:13:34] request.INFO: Matched route "_profiler_search_bar". {"route":"_profiler_search_bar","route_parameters":{"_route":"_profiler_search_bar","_controller":"web_profiler.controller.profiler::searchBarAction"},"request_uri":"http://localhost:8002/_profiler/search_bar?panel=logger","method":"GET"} []
[2019-11-02 12:15:11] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_route":"_profiler","_controller":"web_profiler.controller.profiler::panelAction","token":"804dac"},"request_uri":"http://localhost:8002/_profiler/804dac?panel=logger","method":"GET"} []
[2019-11-02 12:15:11] request.INFO: Matched route "_profiler_search_bar". {"route":"_profiler_search_bar","route_parameters":{"_route":"_profiler_search_bar","_controller":"web_profiler.controller.profiler::searchBarAction"},"request_uri":"http://localhost:8002/_profiler/search_bar?panel=logger","method":"GET"} []
[2019-11-02 12:15:16] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\PagesController::index"},"request_uri":"http://localhost:8002/","method":"GET"} []
[2019-11-02 12:15:16] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-02 12:15:16] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable. Controller "App\Controller\PagesController" does neither exist as service nor as class" at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\http-kernel\Controller\ControllerResolver.php line 88 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable. Controller \"App\\Controller\\PagesController\" does neither exist as service nor as class at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\http-kernel\\Controller\\ControllerResolver.php:88)"} []
[2019-11-02 12:15:17] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"9a9509"},"request_uri":"http://localhost:8002/_wdt/9a9509","method":"GET"} []
[2019-11-02 12:17:44] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\PagesController::index"},"request_uri":"http://localhost:8002/","method":"GET"} []
[2019-11-02 12:17:44] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-02 12:22:18] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\PagesController::index"},"request_uri":"http://localhost:8002/","method":"GET"} []
[2019-11-02 12:22:18] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-02 12:34:11] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\PagesController::index"},"request_uri":"http://localhost:8002/","method":"GET"} []
[2019-11-02 12:34:11] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-02 12:34:11] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable. Controller "App\Controller\PagesController" does neither exist as service nor as class" at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\http-kernel\Controller\ControllerResolver.php line 88 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable. Controller \"App\\Controller\\PagesController\" does neither exist as service nor as class at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\http-kernel\\Controller\\ControllerResolver.php:88)"} []
[2019-11-02 12:34:11] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"74b1ee"},"request_uri":"http://localhost:8002/_wdt/74b1ee","method":"GET"} []
[2019-11-02 12:36:42] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\PagesController::index"},"request_uri":"http://localhost:8002/","method":"GET"} []
[2019-11-02 12:36:42] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-02 12:36:42] php.CRITICAL: Class 'App\Controller\Response' not found {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Class 'App\\Controller\\Response' not found at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\src\\Controller\\PagesController.php:9)"} []
[2019-11-02 12:36:46] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\ClassNotFoundException: "Attempted to load class "Response" from namespace "App\Controller". Did you forget a "use" statement for e.g. "Symfony\Flex\Response", "Symfony\Component\HttpFoundation\Response" or "Symfony\Component\BrowserKit\Response"?" at C:\Users\jean\Desktop\dotdev_lib\symfony\src\Controller\PagesController.php line 9 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\ClassNotFoundException(code: 0): Attempted to load class \"Response\" from namespace \"App\\Controller\".\nDid you forget a \"use\" statement for e.g. \"Symfony\\Flex\\Response\", \"Symfony\\Component\\HttpFoundation\\Response\" or \"Symfony\\Component\\BrowserKit\\Response\"? at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\src\\Controller\\PagesController.php:9)"} []
[2019-11-02 12:36:46] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"5a0c33"},"request_uri":"http://localhost:8002/_wdt/5a0c33","method":"GET"} []
[2019-11-02 13:58:26] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\frontController::index"},"request_uri":"http://localhost:8002/","method":"GET"} []
[2019-11-02 13:58:26] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-02 14:35:36] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\frontController::index"},"request_uri":"http://localhost:8002/","method":"GET"} []
[2019-11-02 14:35:36] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-03 07:16:02] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\frontController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2019-11-03 07:16:03] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-03 07:17:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\frontController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2019-11-03 07:17:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-03 07:17:01] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost:8000/")" at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\http-kernel\EventListener\RouterListener.php line 138 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost:8000/\") at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:138, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:70)"} []
[2019-11-03 07:17:17] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\frontController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2019-11-03 07:17:17] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-03 07:27:03] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\frontController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2019-11-03 07:27:03] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-03 07:27:03] request.CRITICAL: Uncaught PHP Exception Twig\Error\LoaderError: "Unable to find template "some content" (looked into: C:\Users\jean\Desktop\dotdev_lib\symfony/templates, C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\twig-bridge/Resources/views/Form)." at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\twig\twig\src\Loader\FilesystemLoader.php line 250 {"exception":"[object] (Twig\\Error\\LoaderError(code: 0): Unable to find template \"some content\" (looked into: C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony/templates, C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\twig-bridge/Resources/views/Form). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\twig\\twig\\src\\Loader\\FilesystemLoader.php:250)"} []
[2019-11-03 07:27:03] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"0c7372"},"request_uri":"http://localhost:8000/_wdt/0c7372","method":"GET"} []
[2019-11-03 07:27:43] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\frontController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2019-11-03 07:27:43] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-03 07:27:43] request.CRITICAL: Uncaught PHP Exception Twig\Error\LoaderError: "Unable to find template "index.html.twig" (looked into: C:\Users\jean\Desktop\dotdev_lib\symfony/templates, C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\twig-bridge/Resources/views/Form)." at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\twig\twig\src\Loader\FilesystemLoader.php line 250 {"exception":"[object] (Twig\\Error\\LoaderError(code: 0): Unable to find template \"index.html.twig\" (looked into: C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony/templates, C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\twig-bridge/Resources/views/Form). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\twig\\twig\\src\\Loader\\FilesystemLoader.php:250)"} []
[2019-11-03 07:27:44] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"8d6ba8"},"request_uri":"http://localhost:8000/_wdt/8d6ba8","method":"GET"} []
[2019-11-03 07:28:16] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\frontController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2019-11-03 07:28:16] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-03 07:28:17] request.CRITICAL: Uncaught PHP Exception Twig\Error\LoaderError: "Unable to find template "default.index.html.twig" (looked into: C:\Users\jean\Desktop\dotdev_lib\symfony/templates, C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\twig-bridge/Resources/views/Form)." at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\twig\twig\src\Loader\FilesystemLoader.php line 250 {"exception":"[object] (Twig\\Error\\LoaderError(code: 0): Unable to find template \"default.index.html.twig\" (looked into: C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony/templates, C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\twig-bridge/Resources/views/Form). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\twig\\twig\\src\\Loader\\FilesystemLoader.php:250)"} []
[2019-11-03 07:28:17] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"87a056"},"request_uri":"http://localhost:8000/_wdt/87a056","method":"GET"} []
[2019-11-03 07:28:47] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\frontController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2019-11-03 07:28:47] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-03 07:28:47] request.CRITICAL: Uncaught PHP Exception Twig\Error\LoaderError: "Unable to find template "index.html.twig" (looked into: C:\Users\jean\Desktop\dotdev_lib\symfony/templates, C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\twig-bridge/Resources/views/Form)." at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\twig\twig\src\Loader\FilesystemLoader.php line 250 {"exception":"[object] (Twig\\Error\\LoaderError(code: 0): Unable to find template \"index.html.twig\" (looked into: C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony/templates, C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\twig-bridge/Resources/views/Form). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\twig\\twig\\src\\Loader\\FilesystemLoader.php:250)"} []
[2019-11-03 07:28:48] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"8c55ae"},"request_uri":"http://localhost:8000/_wdt/8c55ae","method":"GET"} []
[2019-11-03 07:31:57] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\frontController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2019-11-03 07:31:58] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-03 07:31:58] request.CRITICAL: Uncaught PHP Exception Twig\Error\LoaderError: "Unable to find template "pages/index.html.twig" (looked into: C:\Users\jean\Desktop\dotdev_lib\symfony/templates, C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\twig-bridge/Resources/views/Form)." at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\twig\twig\src\Loader\FilesystemLoader.php line 250 {"exception":"[object] (Twig\\Error\\LoaderError(code: 0): Unable to find template \"pages/index.html.twig\" (looked into: C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony/templates, C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\twig-bridge/Resources/views/Form). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\twig\\twig\\src\\Loader\\FilesystemLoader.php:250)"} []
[2019-11-03 07:31:58] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"12665e"},"request_uri":"http://localhost:8000/_wdt/12665e","method":"GET"} []
[2019-11-03 07:33:15] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\frontController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2019-11-03 07:33:15] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-03 07:33:15] request.CRITICAL: Uncaught PHP Exception Twig\Error\LoaderError: "Unable to find template "pages/index.html.twig" (looked into: C:\Users\jean\Desktop\dotdev_lib\symfony/templates, C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\twig-bridge/Resources/views/Form)." at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\twig\twig\src\Loader\FilesystemLoader.php line 250 {"exception":"[object] (Twig\\Error\\LoaderError(code: 0): Unable to find template \"pages/index.html.twig\" (looked into: C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony/templates, C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\twig-bridge/Resources/views/Form). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\twig\\twig\\src\\Loader\\FilesystemLoader.php:250)"} []
[2019-11-03 07:33:16] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"18b8ba"},"request_uri":"http://localhost:8000/_wdt/18b8ba","method":"GET"} []
[2019-11-03 07:47:11] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\frontController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2019-11-03 07:47:11] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-03 07:47:11] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"0ef519"},"request_uri":"http://localhost:8000/_wdt/0ef519","method":"GET"} []
[2019-11-03 08:26:43] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\frontController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2019-11-03 08:26:43] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-03 08:26:43] request.CRITICAL: Uncaught PHP Exception Twig\Error\SyntaxError: "A template that extends another one cannot include content outside Twig blocks. Did you forget to put the content inside a {% block %} tag?" at C:\Users\jean\Desktop\dotdev_lib\symfony\templates\pages\index.html.twig line 2 {"exception":"[object] (Twig\\Error\\SyntaxError(code: 0): A template that extends another one cannot include content outside Twig blocks. Did you forget to put the content inside a {% block %} tag? at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\templates\\pages\\index.html.twig:2)"} []
[2019-11-03 08:26:44] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"15435e"},"request_uri":"http://localhost:8000/_wdt/15435e","method":"GET"} []
[2019-11-03 08:28:17] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\frontController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2019-11-03 08:28:17] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-03 08:28:18] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"75d777"},"request_uri":"http://localhost:8000/_wdt/75d777","method":"GET"} []
[2019-11-03 08:50:39] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Config\Exception\LoaderLoadException: "[Syntax Error] Expected PlainValue, got ''' at position 7 in method App\Controller\frontController::index() in C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes\../../src/Controller/ (which is being imported from "C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes/annotations.yaml"). Make sure annotations are installed and enabled." at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\config\Loader\FileLoader.php line 166 {"exception":"[object] (Symfony\\Component\\Config\\Exception\\LoaderLoadException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index() in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes\\../../src/Controller/ (which is being imported from \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes/annotations.yaml\"). Make sure annotations are installed and enabled. at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\config\\Loader\\FileLoader.php:166, Doctrine\\Common\\Annotations\\AnnotationException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index(). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\doctrine\\annotations\\lib\\Doctrine\\Common\\Annotations\\AnnotationException.php:42)"} []
[2019-11-03 08:50:39] request.CRITICAL: Exception thrown when handling an exception (Twig\Error\RuntimeError: An exception has been thrown during the rendering of a template ("[Syntax Error] Expected PlainValue, got ''' at position 7 in method App\Controller\frontController::index() in C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes\../../src/Controller/ (which is being imported from "C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes/annotations.yaml"). Make sure annotations are installed and enabled."). at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\twig-bundle\Resources\views\Exception\trace.html.twig line 13) {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"[Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index() in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes\\../../src/Controller/ (which is being imported from \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes/annotations.yaml\"). Make sure annotations are installed and enabled.\"). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\twig-bundle\\Resources\\views\\Exception\\trace.html.twig:13, Symfony\\Component\\Config\\Exception\\LoaderLoadException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index() in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes\\../../src/Controller/ (which is being imported from \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes/annotations.yaml\"). Make sure annotations are installed and enabled. at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\config\\Loader\\FileLoader.php:166, Doctrine\\Common\\Annotations\\AnnotationException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index(). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\doctrine\\annotations\\lib\\Doctrine\\Common\\Annotations\\AnnotationException.php:42)"} []
[2019-11-03 08:50:39] php.CRITICAL: Uncaught Exception: An exception has been thrown during the rendering of a template ("[Syntax Error] Expected PlainValue, got ''' at position 7 in method App\Controller\frontController::index() in C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes\../../src/Controller/ (which is being imported from "C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes/annotations.yaml"). Make sure annotations are installed and enabled."). {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"[Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index() in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes\\../../src/Controller/ (which is being imported from \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes/annotations.yaml\"). Make sure annotations are installed and enabled.\"). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\twig-bundle\\Resources\\views\\Exception\\trace.html.twig:13, Symfony\\Component\\Config\\Exception\\LoaderLoadException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index() in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes\\../../src/Controller/ (which is being imported from \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes/annotations.yaml\"). Make sure annotations are installed and enabled. at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\config\\Loader\\FileLoader.php:166, Doctrine\\Common\\Annotations\\AnnotationException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index(). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\doctrine\\annotations\\lib\\Doctrine\\Common\\Annotations\\AnnotationException.php:42, Symfony\\Component\\Config\\Exception\\LoaderLoadException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index() in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes\\../../src/Controller/ (which is being imported from \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes/annotations.yaml\"). Make sure annotations are installed and enabled. at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\config\\Loader\\FileLoader.php:166, Doctrine\\Common\\Annotations\\AnnotationException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index(). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\doctrine\\annotations\\lib\\Doctrine\\Common\\Annotations\\AnnotationException.php:42)"} []
[2019-11-03 08:50:45] request.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "An exception has been thrown during the rendering of a template ("[Syntax Error] Expected PlainValue, got ''' at position 7 in method App\Controller\frontController::index() in C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes\../../src/Controller/ (which is being imported from "C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes/annotations.yaml"). Make sure annotations are installed and enabled.")." at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\twig-bundle\Resources\views\Exception\trace.html.twig line 13 {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"[Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index() in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes\\../../src/Controller/ (which is being imported from \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes/annotations.yaml\"). Make sure annotations are installed and enabled.\"). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\twig-bundle\\Resources\\views\\Exception\\trace.html.twig:13, Symfony\\Component\\Config\\Exception\\LoaderLoadException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index() in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes\\../../src/Controller/ (which is being imported from \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes/annotations.yaml\"). Make sure annotations are installed and enabled. at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\config\\Loader\\FileLoader.php:166, Doctrine\\Common\\Annotations\\AnnotationException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index(). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\doctrine\\annotations\\lib\\Doctrine\\Common\\Annotations\\AnnotationException.php:42, Symfony\\Component\\Config\\Exception\\LoaderLoadException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index() in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes\\../../src/Controller/ (which is being imported from \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes/annotations.yaml\"). Make sure annotations are installed and enabled. at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\config\\Loader\\FileLoader.php:166, Doctrine\\Common\\Annotations\\AnnotationException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index(). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\doctrine\\annotations\\lib\\Doctrine\\Common\\Annotations\\AnnotationException.php:42)"} []
[2019-11-03 08:50:45] request.CRITICAL: Exception thrown when handling an exception (Twig\Error\RuntimeError: An exception has been thrown during the rendering of a template ("[Syntax Error] Expected PlainValue, got ''' at position 7 in method App\Controller\frontController::index() in C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes\../../src/Controller/ (which is being imported from "C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes/annotations.yaml"). Make sure annotations are installed and enabled."). at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\twig-bundle\Resources\views\Exception\trace.html.twig line 13) {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"[Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index() in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes\\../../src/Controller/ (which is being imported from \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes/annotations.yaml\"). Make sure annotations are installed and enabled.\"). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\twig-bundle\\Resources\\views\\Exception\\trace.html.twig:13, Symfony\\Component\\Config\\Exception\\LoaderLoadException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index() in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes\\../../src/Controller/ (which is being imported from \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes/annotations.yaml\"). Make sure annotations are installed and enabled. at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\config\\Loader\\FileLoader.php:166, Doctrine\\Common\\Annotations\\AnnotationException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index(). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\doctrine\\annotations\\lib\\Doctrine\\Common\\Annotations\\AnnotationException.php:42)"} []
[2019-11-03 08:51:52] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Config\Exception\LoaderLoadException: "[Syntax Error] Expected PlainValue, got ''' at position 7 in method App\Controller\frontController::index() in C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes\../../src/Controller/ (which is being imported from "C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes/annotations.yaml"). Make sure annotations are installed and enabled." at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\config\Loader\FileLoader.php line 166 {"exception":"[object] (Symfony\\Component\\Config\\Exception\\LoaderLoadException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index() in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes\\../../src/Controller/ (which is being imported from \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes/annotations.yaml\"). Make sure annotations are installed and enabled. at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\config\\Loader\\FileLoader.php:166, Doctrine\\Common\\Annotations\\AnnotationException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index(). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\doctrine\\annotations\\lib\\Doctrine\\Common\\Annotations\\AnnotationException.php:42)"} []
[2019-11-03 08:51:52] request.CRITICAL: Exception thrown when handling an exception (Twig\Error\RuntimeError: An exception has been thrown during the rendering of a template ("[Syntax Error] Expected PlainValue, got ''' at position 7 in method App\Controller\frontController::index() in C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes\../../src/Controller/ (which is being imported from "C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes/annotations.yaml"). Make sure annotations are installed and enabled."). at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\twig-bundle\Resources\views\Exception\trace.html.twig line 13) {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"[Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index() in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes\\../../src/Controller/ (which is being imported from \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes/annotations.yaml\"). Make sure annotations are installed and enabled.\"). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\twig-bundle\\Resources\\views\\Exception\\trace.html.twig:13, Symfony\\Component\\Config\\Exception\\LoaderLoadException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index() in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes\\../../src/Controller/ (which is being imported from \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes/annotations.yaml\"). Make sure annotations are installed and enabled. at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\config\\Loader\\FileLoader.php:166, Doctrine\\Common\\Annotations\\AnnotationException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index(). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\doctrine\\annotations\\lib\\Doctrine\\Common\\Annotations\\AnnotationException.php:42)"} []
[2019-11-03 08:51:52] php.CRITICAL: Uncaught Exception: An exception has been thrown during the rendering of a template ("[Syntax Error] Expected PlainValue, got ''' at position 7 in method App\Controller\frontController::index() in C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes\../../src/Controller/ (which is being imported from "C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes/annotations.yaml"). Make sure annotations are installed and enabled."). {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"[Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index() in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes\\../../src/Controller/ (which is being imported from \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes/annotations.yaml\"). Make sure annotations are installed and enabled.\"). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\twig-bundle\\Resources\\views\\Exception\\trace.html.twig:13, Symfony\\Component\\Config\\Exception\\LoaderLoadException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index() in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes\\../../src/Controller/ (which is being imported from \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes/annotations.yaml\"). Make sure annotations are installed and enabled. at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\config\\Loader\\FileLoader.php:166, Doctrine\\Common\\Annotations\\AnnotationException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index(). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\doctrine\\annotations\\lib\\Doctrine\\Common\\Annotations\\AnnotationException.php:42, Symfony\\Component\\Config\\Exception\\LoaderLoadException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index() in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes\\../../src/Controller/ (which is being imported from \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes/annotations.yaml\"). Make sure annotations are installed and enabled. at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\config\\Loader\\FileLoader.php:166, Doctrine\\Common\\Annotations\\AnnotationException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index(). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\doctrine\\annotations\\lib\\Doctrine\\Common\\Annotations\\AnnotationException.php:42)"} []
[2019-11-03 08:51:58] request.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "An exception has been thrown during the rendering of a template ("[Syntax Error] Expected PlainValue, got ''' at position 7 in method App\Controller\frontController::index() in C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes\../../src/Controller/ (which is being imported from "C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes/annotations.yaml"). Make sure annotations are installed and enabled.")." at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\twig-bundle\Resources\views\Exception\trace.html.twig line 13 {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"[Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index() in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes\\../../src/Controller/ (which is being imported from \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes/annotations.yaml\"). Make sure annotations are installed and enabled.\"). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\twig-bundle\\Resources\\views\\Exception\\trace.html.twig:13, Symfony\\Component\\Config\\Exception\\LoaderLoadException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index() in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes\\../../src/Controller/ (which is being imported from \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes/annotations.yaml\"). Make sure annotations are installed and enabled. at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\config\\Loader\\FileLoader.php:166, Doctrine\\Common\\Annotations\\AnnotationException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index(). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\doctrine\\annotations\\lib\\Doctrine\\Common\\Annotations\\AnnotationException.php:42, Symfony\\Component\\Config\\Exception\\LoaderLoadException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index() in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes\\../../src/Controller/ (which is being imported from \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes/annotations.yaml\"). Make sure annotations are installed and enabled. at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\config\\Loader\\FileLoader.php:166, Doctrine\\Common\\Annotations\\AnnotationException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index(). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\doctrine\\annotations\\lib\\Doctrine\\Common\\Annotations\\AnnotationException.php:42)"} []
[2019-11-03 08:51:58] request.CRITICAL: Exception thrown when handling an exception (Twig\Error\RuntimeError: An exception has been thrown during the rendering of a template ("[Syntax Error] Expected PlainValue, got ''' at position 7 in method App\Controller\frontController::index() in C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes\../../src/Controller/ (which is being imported from "C:\Users\jean\Desktop\dotdev_lib\symfony\config/routes/annotations.yaml"). Make sure annotations are installed and enabled."). at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\twig-bundle\Resources\views\Exception\trace.html.twig line 13) {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"[Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index() in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes\\../../src/Controller/ (which is being imported from \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes/annotations.yaml\"). Make sure annotations are installed and enabled.\"). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\twig-bundle\\Resources\\views\\Exception\\trace.html.twig:13, Symfony\\Component\\Config\\Exception\\LoaderLoadException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index() in C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes\\../../src/Controller/ (which is being imported from \"C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\config/routes/annotations.yaml\"). Make sure annotations are installed and enabled. at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\config\\Loader\\FileLoader.php:166, Doctrine\\Common\\Annotations\\AnnotationException(code: 0): [Syntax Error] Expected PlainValue, got ''' at position 7 in method App\\Controller\\frontController::index(). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\doctrine\\annotations\\lib\\Doctrine\\Common\\Annotations\\AnnotationException.php:42)"} []
[2019-11-03 08:52:44] request.INFO: Matched route "app_front_index". {"route":"app_front_index","route_parameters":{"_route":"app_front_index","_controller":"App\\Controller\\frontController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2019-11-03 08:52:44] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-03 08:52:44] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"1f730f"},"request_uri":"http://localhost:8000/_wdt/1f730f","method":"GET"} []
[2019-11-03 08:53:03] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"App\\Controller\\frontController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2019-11-03 08:53:03] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-03 08:53:03] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"e0243d"},"request_uri":"http://localhost:8000/_wdt/e0243d","method":"GET"} []
[2019-11-03 09:19:51] request.INFO: Matched route "posts". {"route":"posts","route_parameters":{"_route":"posts","_controller":"App\\Controller\\PostsController::index"},"request_uri":"http://localhost:8000/posts","method":"GET"} []
[2019-11-03 09:19:51] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-03 09:19:51] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"391d0c"},"request_uri":"http://localhost:8000/_wdt/391d0c","method":"GET"} []
[2019-11-03 09:20:53] request.INFO: Matched route "posts". {"route":"posts","route_parameters":{"_route":"posts","_controller":"App\\Controller\\PostsController::index"},"request_uri":"http://localhost:8000/posts","method":"GET"} []
[2019-11-03 09:20:53] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-03 09:20:53] request.CRITICAL: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\ControllerDoesNotReturnResponseException: "The controller must return a "Symfony\Component\HttpFoundation\Response" object but it returned a string ("<!DOCTYPE html> <html>     <head>         <meta charset="UTF-8">         <title>Hello PostsController!</title>             </head>     <body>         <style>     .example-wrapper { margin: 1em auto; max-width: 800px; width: 95%; font: 18px/1.5 sans-serif;...")." at C:\Users\jean\Desktop\dotdev_lib\symfony\src\Controller\PostsController.php line 19 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\ControllerDoesNotReturnResponseException(code: 0): The controller must return a \"Symfony\\Component\\HttpFoundation\\Response\" object but it returned a string (\"<!DOCTYPE html>\n<html>\n    <head>\n        <meta charset=\"UTF-8\">\n        <title>Hello PostsController!</title>\n            </head>\n    <body>\n        <style>\n    .example-wrapper { margin: 1em auto; max-width: 800px; width: 95%; font: 18px/1.5 sans-serif;...\"). at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\src\\Controller\\PostsController.php:19)"} []
[2019-11-03 09:20:54] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"a28e8c"},"request_uri":"http://localhost:8000/_wdt/a28e8c","method":"GET"} []
[2019-11-03 09:21:54] request.INFO: Matched route "posts". {"route":"posts","route_parameters":{"_route":"posts","_controller":"App\\Controller\\PostsController::index"},"request_uri":"http://localhost:8000/posts","method":"GET"} []
[2019-11-03 09:21:55] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-03 09:21:55] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"9455e3"},"request_uri":"http://localhost:8000/_wdt/9455e3","method":"GET"} []
[2019-11-03 09:29:50] request.INFO: Matched route "_profiler_open_file". {"route":"_profiler_open_file","route_parameters":{"_route":"_profiler_open_file","_controller":"web_profiler.controller.profiler::openAction"},"request_uri":"http://localhost:8000/_profiler/open?file=C%3A%2FUsers%2Fjean%2FDesktop%2Fdotdev_lib%2Fsymfony%2Fsrc%2FController%2FPostsController.php&line=0","method":"GET"} []
[2019-11-03 09:29:50] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "The file "C:/Users/jean/Desktop/dotdev_lib/symfony/src/Controller/PostsController.php" cannot be opened." at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\web-profiler-bundle\Controller\ProfilerController.php line 350 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): The file \"C:/Users/jean/Desktop/dotdev_lib/symfony/src/Controller/PostsController.php\" cannot be opened. at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\web-profiler-bundle\\Controller\\ProfilerController.php:350)"} []
[2019-11-03 09:29:58] request.INFO: Matched route "_profiler_open_file". {"route":"_profiler_open_file","route_parameters":{"_route":"_profiler_open_file","_controller":"web_profiler.controller.profiler::openAction"},"request_uri":"http://localhost:8000/_profiler/open?file=C%3A%2FUsers%2Fjean%2FDesktop%2Fdotdev_lib%2Fsymfony%2Ftemplates%2Fposts%2Findex.html.twig&line=0","method":"GET"} []
[2019-11-03 09:29:58] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "The file "C:/Users/jean/Desktop/dotdev_lib/symfony/templates/posts/index.html.twig" cannot be opened." at C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\web-profiler-bundle\Controller\ProfilerController.php line 350 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): The file \"C:/Users/jean/Desktop/dotdev_lib/symfony/templates/posts/index.html.twig\" cannot be opened. at C:\\Users\\jean\\Desktop\\dotdev_lib\\symfony\\vendor\\symfony\\web-profiler-bundle\\Controller\\ProfilerController.php:350)"} []
[2019-11-03 09:35:25] request.INFO: Matched route "posts". {"route":"posts","route_parameters":{"_route":"posts","_controller":"App\\Controller\\PostsController::index"},"request_uri":"http://localhost:8000/posts","method":"GET"} []
[2019-11-03 09:35:25] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-11-03 09:35:25] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"d2a851"},"request_uri":"http://localhost:8000/_wdt/d2a851","method":"GET"} []
[2019-11-03 11:04:03] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.en.xlf {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"C:\\\\wamp64\\\\www\\\\dotdev_lib\\\\symfony\\\\vendor\\\\symfony\\\\config\\\\Resource\\\\FileResource.php\",\"line\":65,\"trace\":[{\"file\":\"C:\\\\wamp64\\\\www\\\\dotdev_lib\\\\symfony\\\\vendor\\\\symfony\\\\config\\\\Resource\\\\SelfCheckingResourceChecker.php\",\"line\":34,\"function\":\"isFresh\",\"class\":\"Symfony\\\\Component\\\\Config\\\\Resource\\\\FileResource\",\"type\":\"->\"}],\"count\":1})"} []
[2019-11-03 11:04:03] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.af.xlf [] []
[2019-11-03 11:04:03] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.ar.xlf [] []
[2019-11-03 11:04:03] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.az.xlf [] []
[2019-11-03 11:04:03] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.be.xlf [] []
[2019-11-03 11:04:03] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.bg.xlf [] []
[2019-11-03 11:04:03] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.ca.xlf [] []
[2019-11-03 11:04:03] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.cs.xlf [] []
[2019-11-03 11:04:03] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.cy.xlf [] []
[2019-11-03 11:04:03] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.da.xlf [] []
[2019-11-03 11:04:04] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.de.xlf [] []
[2019-11-03 11:04:04] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.el.xlf [] []
[2019-11-03 11:04:04] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.es.xlf [] []
[2019-11-03 11:04:04] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.et.xlf [] []
[2019-11-03 11:04:04] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.eu.xlf [] []
[2019-11-03 11:04:04] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.fa.xlf [] []
[2019-11-03 11:04:04] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.fi.xlf [] []
[2019-11-03 11:04:04] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.fr.xlf [] []
[2019-11-03 11:04:04] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.gl.xlf [] []
[2019-11-03 11:04:04] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.he.xlf [] []
[2019-11-03 11:04:04] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.hr.xlf [] []
[2019-11-03 11:04:04] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.hu.xlf [] []
[2019-11-03 11:04:04] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.hy.xlf [] []
[2019-11-03 11:04:04] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.id.xlf [] []
[2019-11-03 11:04:04] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.it.xlf [] []
[2019-11-03 11:04:04] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.ja.xlf [] []
[2019-11-03 11:04:04] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.lb.xlf [] []
[2019-11-03 11:04:04] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.lt.xlf [] []
[2019-11-03 11:04:04] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.lv.xlf [] []
[2019-11-03 11:04:04] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.mn.xlf [] []
[2019-11-03 11:04:04] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.nb.xlf [] []
[2019-11-03 11:04:05] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.nl.xlf [] []
[2019-11-03 11:04:05] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.nn.xlf [] []
[2019-11-03 11:04:05] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.no.xlf [] []
[2019-11-03 11:04:05] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.pl.xlf [] []
[2019-11-03 11:04:05] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.pt.xlf [] []
[2019-11-03 11:04:05] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.pt_BR.xlf [] []
[2019-11-03 11:04:05] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.ro.xlf [] []
[2019-11-03 11:04:05] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.ru.xlf [] []
[2019-11-03 11:04:05] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.sk.xlf [] []
[2019-11-03 11:04:05] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.sl.xlf [] []
[2019-11-03 11:04:05] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.sq.xlf [] []
[2019-11-03 11:04:05] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.sr_Cyrl.xlf [] []
[2019-11-03 11:04:05] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.sr_Latn.xlf [] []
[2019-11-03 11:04:05] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.sv.xlf [] []
[2019-11-03 11:04:05] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.th.xlf [] []
[2019-11-03 11:04:05] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.tl.xlf [] []
[2019-11-03 11:04:05] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.tr.xlf [] []
[2019-11-03 11:04:05] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.uk.xlf [] []
[2019-11-03 11:04:05] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.vi.xlf [] []
[2019-11-03 11:04:05] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.zh_CN.xlf [] []
[2019-11-03 11:04:05] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\validator\Resources\translations\validators.zh_TW.xlf [] []
[2019-11-03 11:04:05] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\vendor\symfony\security-core\Resources\translations\security.pt_PT.xlf [] []
[2019-11-03 11:04:06] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\config\routes\dev\twig.yaml [] []
[2019-11-03 11:04:06] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\jean\Desktop\dotdev_lib\symfony\config\routes\dev\twig.yaml [] []
